core/controller.go

```diff
@@ -38,12 +41,20 @@ func (c *Controller) Activate() *resources.ActivateResponse {
 	return &resources.ActivateResponse{Implements: []string{"VolumeDriver"}}
```

Comment:
should become:
return resources.ActivateResponse{Implements: []string{"VolumeDriver"}}